package.json

```diff
@@ -52,7 +52,7 @@
 					"${workspaceFolder}"
 				],
 				"pattern": {
-					"regexp": "^(\\w+) at (.+):(\\d+):(\\d+): (.+)$",
+					"regexp": "^(Error|Warning) OD\\d+ at (.+):(\\d+):(\\d+): (.+)$",
```

Comment:
You could put `(OD\\d+)` in parentheses and add a `"code":` property for it (and update the other indices).

Reply:
Nice, thanks.